bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryConnectorException.java

```diff
@@ -54,4 +55,28 @@ public InvalidSchemaException(String message, Throwable t) {
       super(BIGQUERY_INVALID_SCHEMA, message, t);
     }
   }
+
+  /** Exception if query pushdown to BigQuery fails */
+  public static class PushdownException extends BigQueryConnectorException {
```

Comment:
Please move exceptions to be top level classes

Reply:
done